addons/html_builder/static/src/plugins/background_option/background_position_option_plugin.js

```diff
@@ -31,12 +34,17 @@ export class BackgroundTypeAction extends BuilderAction {
         editingElement.style.setProperty("background-position", "");
         editingElement.style.setProperty(
             "background-size",
-            value !== "repeat-pattern" ? "" : "100px"
+            value !== "repeat-pattern" ? "" : "100px, cover"
         );
     }
     isApplied({ editingElement, value }) {
-        const hasElRepeatStyle = getComputedStyle(editingElement).backgroundRepeat === "repeat";
-        return value === "repeat-pattern" ? hasElRepeatStyle : !hasElRepeatStyle;
+        const bgRepeat = getComputedStyle(editingElement).backgroundRepeat;
```

Comment:
Could you check this here an other places too, you can get value of `backgroundRepeat` or `backgroundSize` (in other places) directly from `.style` of an element, this could help reduce a lot of looping and array jugglings what do you think ? :thinking: